LICENSE

```diff
@@ -1,27 +1,17 @@
-Copyright (c) 2015-2017, conda-forge
```

Comment:
Please don't change this file. It is the license file for this repo not the recipe being added.